--- a/arch/arm64/boot/dts/allwinner/sun50i-h616-orangepi-zero.dtsi	2024-10-19 02:33:42.965531832 +0100
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616-orangepi-zero.dtsi	2024-10-19 03:36:08.655907275 +0100
@@ -26,16 +26,16 @@
 		compatible = "gpio-leds";
 
 		led-0 {
-			function = LED_FUNCTION_POWER;
+			function = LED_FUNCTION_STATE;
 			color = <LED_COLOR_ID_RED>;
 			gpios = <&pio 2 12 GPIO_ACTIVE_HIGH>; /* PC12 */
-			default-state = "on";
 		};
 
 		led-1 {
-			function = LED_FUNCTION_STATUS;
+			function = LED_FUNCTION_POWER;
 			color = <LED_COLOR_ID_GREEN>;
 			gpios = <&pio 2 13 GPIO_ACTIVE_HIGH>; /* PC13 */
+			default-state = "on";
 		};
 	};
 
